plugin.video.opentakserver/addon.py

```diff
@@ -0,0 +1,173 @@
+import datetime
+import os
+import sys
+import uuid
+from urllib.parse import urlencode, parse_qsl
+import requests
+
+import xbmc
+import xbmcgui
+import xbmcplugin
+import xbmcvfs
+import xbmcaddon
+from xbmcaddon import Addon
+from xbmcvfs import translatePath
+
+# Get the plugin url in plugin:// notation.
+URL = sys.argv[0]
+# Get a plugin handle as an integer number.
+HANDLE = int(sys.argv[1])
+# Get addon base path
+ADDON_PATH = translatePath(Addon().getAddonInfo('path'))
+ICONS_DIR = os.path.join(ADDON_PATH, 'resources', 'images', 'icons')
+FANART_DIR = os.path.join(ADDON_PATH, 'resources', 'images', 'fanart')
+__addon__ = xbmcaddon.Addon(id='plugin.video.opentakserver')
+__addondir__ = xbmcvfs.translatePath( __addon__.getAddonInfo('profile'))
+
+csrf_token = None
+
+
+def get_auth_token_from_file():
+    f = xbmcvfs.File(os.path.join(__addondir__, "auth_token"), 'r')
+    auth_token = f.read()
+    f.close()
+    return auth_token
+
+
+def get(url):
+    auth_token = get_auth_token_from_file()
+
+    r = requests.get(url, params={'auth_token': auth_token}, verify=False)
```

Comment:
I think `verify=False` should be configurable: some might actually have certificates and then they don't want to just ignore invalid ones?

Reply:
I've updated the PR with this option